benchmark/configs/async_fl/async_fl.yml

```diff
@@ -31,27 +32,23 @@ setup_commands:
 job_conf:
     - job_name: femnist                   # Generate logs under this folder: log_path/job_name/time_stamp
     - log_path: $FEDSCALE_HOME/benchmark # Path of log files
-    - num_participants: 800                      # Number of participants per round, we use K=100 in our paper, large K will be much slower
+    - num_participants: 20                      # Number of participants per round, we use K=100 in our paper, large K will be much slower
```

Comment:
The comment for this param might need to be updated for async. How do we decide what to use for num_participants since there's no notion of rounds in async?